src/resizable.directive.ts

```diff
@@ -416,7 +422,7 @@ export class ResizableDirective implements OnInit, OnChanges, OnDestroy {
       this.mousemove
     ).pipe(
       tap(({ event }) => {
-        if (currentResize) {
+        if (currentResize && event.defaultPrevented) {
```

Comment:
Should this be `!event.defaultPrevented`?

Reply:
I investigated in this topic a bit more and found put that I just blindly copypasted from some stackoverflow 😅
and this field does not actually reflect passive eventlistener

The truth is there is no field to say that, so the only workaround is to add try-catch